src/components/o-modal/_preview.html

```diff
@@ -0,0 +1,80 @@
+<!--Please create here a HTML example by using just default HTML tags-->
+
+<button onclick="openModal()">Open modal</button>
+
+<axa-modal class="axa-modal-preview">
+  <h2>Prämie berechnen</h2>
+  <p><strong>Mehr als 80%</strong> unserer Kunden kombinieren die Hausratversicherung mit einer Privathaftpflichtversicherung.</p>
+  <p>Was möchten Sie?</p>
+
+  <section class="grid">
```

Comment:
würde für das preview nicht selber classe schreiben sondern `axa-container` oder axa griod verwedenen

Reply:
@raphaellueckl please use `axa-container` instead of creating a custom grid class